src/widgets/NftDetailWidget/subcomponents/ConnectedNftDetailWidget/ConnectedNftDetailWidget.tsx

```diff
@@ -60,7 +60,7 @@ const ConnectedNftDetailWidget: FC<ConnectedNftDetailWidgetProps> = ({ library,
           />
           <NftDetailSaleInfo
             price={BigNumber.from('0')}
-            symbol={tokenMeta.symbol}
+            symbol={tokenMeta.name}
```

Comment:
What is the reason for this change?

Reply:
The CollectionTokenInfo from airswap package will not have `symbol` anymore, just `name`.

Reply:
Where do we plan to get the symbol from moving forwards?

Reply:
Only erc-20 (`TokenInfo`) token will have a symbol